base/exports.jl

```diff
@@ -994,7 +994,6 @@ export
 # loading source files
     __precompile__,
     evalfile,
-    include,
```

Comment:
This wasn't supposed to be exported – it causes binding overwrite warnings when you try to extend the system image with compile all.